src/components/HomePage/HomePage.tsx

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+export const HomePage: React.FC = () => (
+  <h1 className="title">Home Page</h1>
```

Comment:
It seems that you are using a className 'title' but no styles or CSS imports are provided. Make sure to include the necessary CSS file or styled-components for styling.

Reply:
CSS is imported into src/index.tsx. It takes styles from bulma